src/jdk.incubator.foreign/share/classes/jdk/internal/foreign/abi/Binding.java

```diff
@@ -215,11 +215,11 @@
                     methodType(MemoryAddress.class, long.class));
             MH_BASE_ADDRESS = lookup.findVirtual(MemorySegment.class, "address",
                     methodType(MemoryAddress.class));
-            MH_COPY_BUFFER = lookup.findStatic(Binding.class, "copyBuffer",
+            MH_COPY_BUFFER = lookup.findStatic(Binding.Copy.class, "copyBuffer",
```

Comment:
I'm assuming these are only really required in the specialize() method of the various subclasses - should we move those constants there too?

Reply:
We can, though we end up duplicating the try/catch & lookup code if we do, or would have to add some helper method, for both findStatic and findVirtual. That's why I held off on doing that.